ecs_grafana.tf

```diff
@@ -148,3 +148,69 @@ resource "aws_security_group_rule" "allow_loadbalancer_ingress_grafana_http" {
   security_group_id        = aws_security_group.grafana[0].id
   source_security_group_id = aws_security_group.monitoring[0].id
 }
+
+resource "aws_iam_role" "grafana" {
+  count              = local.is_management_env ? 1 : 0
+  name               = "grafana"
+  assume_role_policy = data.aws_iam_policy_document.grafana[local.primary_role_index].json
+  tags               = merge(local.tags, { Name = "grafana" })
+}
+
+data "aws_iam_policy_document" "grafana" {
```

Comment:
General naming convention for these types of policy docs is `grafana_assume_role` to differentiate it from other policies attached to the role.

Reply:
Resolved